spec/OneClickInstall_spec.rb

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env rspec
+
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "spec_helper"
+
+require "yast"
+
+Yast.import "OneClickInstall"
+
+describe "Yast::OneClickInstall" do
+  subject { Yast::OneClickInstall }
+  let(:filepath) { File.join(DATA_PATH, "vim.ymp") }
+
+  describe "#xml_from_file" do
+    context "ymp file parsed" do
+      it "parses the XML file" do
+        doc = subject.xml_from_file(filepath)
+        expect(doc).to be_instance_of(REXML::Document)
+        output = ""
+        doc.write(output)
+        expect(output).to include("vim")
+      end
+    end
+  end
+
+  describe "#xpath_match" do
```

Comment:
this method is just a wrapper of a core function, I think here we could initialize the elements which we expect to be returned (as described [here](https://github.com/yast/yast-metapackage-handler/blob/master/src/modules/OneClickInstall.rb#L154) and match array with actual result, instead of calling the same library in the code and in the validation.